### PR TITLE
Import news

### PR DESCRIPTION
**Pull request by melish**

### CODE REVIEW

File: dv/management/commands/import_news.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ENDPOINT = 'http://eeagrants.org/rest/articles?page={}'
+
+
+class Command(BaseCommand):
+    help = 'Import the file given as argument'
+
+    def handle(self, *args, **options):
+        page = 0
+        data = []
+        News.objects.all().delete()
```
Comment (xlotlu) [inline dv/management/commands/import_news.py]: if remote sorts items by updated desc, we can skip deleting everything here...

File: dv/management/commands/import_news.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                data = json.loads(url.read().decode())['posts']
+                print('Importing {} news from page {}'.format(len(data), page))
+                for item in data:
+                    self._save(item)
+                page += 1
+
+    def _save(self, item):
+        try:
+            self.stdout.write('Importing post %s' % item['link'])
+            news, created = News.objects.get_or_create(link=item['link'])
```
Comment (xlotlu) [inline dv/management/commands/import_news.py]: ... and if not created, and no change in updated, raise something here that stops everything

File: dv/models.py [hunk 3]
```diff
@@ -562,3 +562,22 @@ class Labels:
     programme = models.ForeignKey(Programme, null=True)
     # TODO: this structure smells
     project = models.ForeignKey(Project, null=True)
+
+
+class News(models.Model):
+    title = models.TextField(null=False, blank=False)
+    link = models.URLField(max_length=2000, null=False, blank=False)
```
Comment (xlotlu) [inline dv/models.py]: TODO: dbindex

File: dv/management/commands/import_news.py (hunk 1 above)
Comment (melish) [inline dv/management/commands/import_news.py]: I know, but not worth to do, it takes a few seconds to import all. Except when endpoint fails with http 500 or timeout

File: dv/models.py (hunk 3 above)
Comment (melish) [inline dv/models.py]: Already indexed by project and programme fk

**Pull request merged**